production_cluster/kibana_ssl/generate-self-signed-cert.sh

```diff
@@ -9,4 +9,5 @@ then
     exit
 else
     openssl req -x509 -batch -nodes -days 365 -newkey rsa:2048 -keyout key.pem -out cert.pem
+    chown -R 1000:1000 .
```

Comment:
Please be specific with those files who need the chown.

Reply:
The function is modified to assign the permissions to all the .pem files in the directory